app/controllers/articles_controller.rb

```diff
@@ -72,6 +73,10 @@ def eds_init
       field.label = 'Abstract'
     end
 
+    config.add_search_field('series') do |field|
+      field.label = 'Series'
```

Comment:
Do we know if series is a supported search type from the EDS gem?  Looking over the [code in master](https://github.com/ebsco/edsapi-ruby/blob/71eddcda215cbcf4455dc5758d04b86ff9406af9/lib/ebsco/eds/options.rb#L344-L368), it doesn't seem like it is?